pbiviz.json

```diff
@@ -4,7 +4,7 @@
     "displayName": "Gantt",
     "guid": "Gantt1448688115699",
     "visualClassName": "Gantt",
-    "version": "1.9.0",
+    "version": "1.10.0",
```

Comment:
Everything is nice bu we need to update version in `package.json` as well

Reply:
Ok